src/components/FileGridWithCart/FileGridView.js

```diff
@@ -0,0 +1,204 @@
+import React, { useRef, useEffect } from 'react';
```

Comment:
Can you rename the component that makes more sense without looking into the foldercomponents